packages/test/mocha-test-setup/README.md

```diff
@@ -1,4 +1,4 @@
-# @fluidframework/mocha-test-setup
+# @fluid-internal/mocha-test-setup
 
```

Comment:
While we're here...

```suggestion

<!-- AUTO-GENERATED-CONTENT:START (README_PACKAGE_SCOPE_NOTICE:packageJsonPath=./package.json) --><!-- AUTO-GENERATED-CONTENT:END -->

```

That way it's a bit more obvious to folks looking at the package on npm or wherever that this isn't intended for external use.

Reply:
Very cool, I didn't even know about this as an option!

Reply:
We've got a fair number of useful pragmas. Most of them are documented here: https://github.com/microsoft/FluidFramework/blob/main/tools/markdown-magic/README.md#transforms (though, notably, not the one I recommended here... I'll fix that...).

We'll want to update our namespace definitions for these if/when we update them.

Reply:
#19763